scripts/date_schedules.py

```diff
@@ -15,6 +17,15 @@
 def make_url(d):
 	return BASE_URL + str(d)
 
+def download_json_with_retry(d):
+	for i in range (3): 
+		try:
+			return download_json(d)
+		except HTTPError as e:
+		#reciving error e, retrying
```

Comment:
receiving

Reply:
Turn this comment into a print(). You'll also need to re-raise e if it's the last time through the loop or else you'll have silent data loss.